debian/changelog

```diff
@@ -1,29 +1,32 @@
-cloud-init (23.1-0ubuntu0~20.04.2) UNRELEASED; urgency=medium
+cloud-init (23.1.1-0ubuntu0~20.04.2) UNRELEASED; urgency=medium
```

Comment:
As mentioned in  the Jammy PR related to this changeset. Since we know `build-package` doesn't like current quilt patch states which don't apply cleanly to cloud-init-23.1.1.orig.tar.gz, let us increment the version to 23.1.2 (which will be the next SRU'd version anyway, or greater). This allows build-package to function on this branch creating it's own local cloud-init-23.1.2.orig.tar.gz from the current branch and avoid quilt patch 'fuzz' for development testing.
```suggestion
cloud-init (23.1.2-0ubuntu0~20.04.1) UNRELEASED; urgency=medium
```

```suggestion
cloud-init (23.1.1-0ubuntu0~20.04.2) UNRELEASED; urgency=medium
```